### ISSUE TITLE
Mail notifications bugged

### ISSUE THREAD

**Issue by Voczi** [gyunaev/birdtray#414]
**OS**: Windows 10 x64 v2004
**Birdtray version**: 1.8.1
**Thunderbird version**: 78.3.1
**Birdtray origin**: From installer
**Qt version**: _NA_ 

**Description**
Pressing on a mail notification does not bring the window up. You would instead have to manually open Thunderbird by double-clicking on the Birdtray's tray icon and it would then, together with an attached Thunderbird window, also launch X amount of detached Thunderbird windows. (It would be reasonable to assume that amount X depends on how many times you pressed the notification.)

**Expected behavior**
A click on the notification should bring the Thunderbird window up immediately and not start the windows "hidden" as it does right now.

**To Reproduce**
1. Receive mail.
2. Click on the title of the mail notification. (Unsure if you have to do so in order to open the mail received, but the title is marked as hypertext.)
3. No Thunderbird windows pop up, until you double-click on the Birdtray tray icon to bring up Thunderbird manually.

**Additional context**
This only happens on occasion. Most of the time, the Thunderbird window pops up as it should do.

**Comment by gyunaev** [comment]
Duplicate of #370

**Comment by Abestanis** [comment]
I don't think this is a duplicate. This bug is about opening Thunderbird windows from the notification that Thunderbird displays for new mail.

**Comment by Abestanis** [comment]
I tried this and when I click on the notification, the main Thunderbird window opens up and displays the mail.
You were describing multiple Thunderbird windows, maybe that is the issue. Do you have some setting enabled in Thunderbird that opens the mail in a new window? If so, where is that setting, I couldn't find it.

**Comment by Voczi** [comment]
> 
> 
> Do you have some setting enabled in Thunderbird that opens the mail in a new window?

Nope. I just have it set to showing notifications when messages arrive.

Thunderbird just stays hidden for some reason (not even on the taskbar, or anything) when you click on the notification. I am unsure if Birdtray has anything to do with it, because it can't even hide Thunderbird windows that fast. (The windows don't even show up, but the processes are in task manager.)

**Comment by Abestanis** [comment]
So even with Birdtray not installed, if you click on the notification, Thunderbird will create a new process and window when clicking on the notification? On my machine, Thunderbird does not behave in that way at all.

Is it possible that you installed a Thunderbird Plugin that changes the behavior of the notification?
Otherwise, would you be able to describe what settings you changed when starting from a fresh installation of Thunderbird?

**Comment by gyunaev** [comment]
I suggest we close this issue unless someone else reports it.

**Comment by Voczi** [comment]
> 
> 
> So even with Birdtray not installed, if you click on the notification, Thunderbird will create a new process and window when clicking on the notification? On my machine, Thunderbird does not behave in that way at all.

Oh no, that's not what I meant, sorry. The process will still be the same. 

> 
> Is it possible that you installed a Thunderbird Plugin that changes the behavior of the notification?

Nope, no plugins.

>
> Otherwise, would you be able to describe what settings you changed when starting from a fresh installation of Thunderbird?

Only modifications I have done related to this is turning the tray icon, as well as the sound notification, off for new messages. And then I disabled the message content display in new notifications.

I received a mail notification just a couple of minutes ago and it worked fine. I think this behavior only happens very rarely, and under special conditions.
